### PR TITLE
Added Ansible UTM Info Module for managing reverse proxy profile entries info state

### PR DESCRIPTION
**Pull request by maunzCache**
##### SUMMARY
This PR will add a new utm info module which allows the management of reverse proxy profile entries of the Sophos UTM. This module integrates with the existing utm_utils .

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
- module/web_infrastructure/sophos_utm/utm_proxy_profile_info.py

### CODE REVIEW

**Comment by MatrixCrawler** [comment]
shipit

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+---
+module: utm_proxy_profile_info
+
+author:
+    - Stephan Schwarz (@stearz)
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py:21]: This should be your name

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py (hunk 1 above)
Comment (maunzCache) [reply lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py:21]: I'm not the author. This code was done by me working colleague so he deserves the honor. Just committed it to github for him.
@stearz Any opinion on that request?

**Comment by getjack** [comment]
I am very sorry but i deleted the remote repo and i didn't check the PRs beforehand. Will redo the PRs and fix it. Sorry for the overhead :(

**Pull request closed without merging**